prime-router/src/main/kotlin/fhirengine/engine/FHIRTranslator.kt

```diff
@@ -163,6 +171,18 @@ class FHIRTranslator(
             topic = message.topic
         )
 
+        val builder = ReportStreamItemEventBuilder(
```

Comment:
We should prefer the style that is shown in FHIRDestinationFilter where we pass in the builder directly to `sendItemEvent` (so we remain consistent):

```
reportEventService.sendItemEvent(
            eventName = ReportStreamEventName.ITEM_ROUTED,
            childReport = report,
            pipelineStepName = TaskAction.destination_filter
        ) {
            parentReportId(queueMessage.reportId)
            params(
                mapOf(
                ReportStreamEventProperties.RECEIVER_NAME to receiver.fullName,
                ReportStreamEventProperties.BUNDLE_DIGEST
                    to bundleDigestExtractor.generateDigest(bundle)
            )
            )
            trackingId(bundle)
        }
```

Reply:
Should be in there now.